tests/init-global/pos/i18628_3.scala

```diff
@@ -6,6 +6,6 @@ object Test:
   class Box(val x: Int)
 
   def recur(a: => Box, b: => Box): Int =
-    a.x + recur(a: @widen(5), b: @widen(5)) + b.x // warn // warn
+    a.x + recur(a: @widen(5), b: @widen(5)) + b.x
 
   recur(Box(1), Box(2))
```

Comment:
We can drop this test, it's the same as the other one. The `widen` annotation does not have a meaning in the new analysis anymore.